chunky/src/java/se/llbit/chunky/model/AABBModel.java

```diff
@@ -14,6 +14,17 @@
  */
 public abstract class AABBModel implements BlockModel {
 
+  /**
+   * Different UV mapping methods.
+   *  - None: No change in mapping
+   *  - ROTATE_90: Rotate 90 degrees clockwise
+   *  - ROTATE_180: Rotate 180 degrees
+   *  - ROTATE_270: Rotate 270 degrees clockwise (90 degrees counterclockwise)
+   *  - FLIP_U: Flip along the X axis (u = 1 - u)
+   *  - FLIP_V: Flip along the Y axis (v = 1 - v)
+   *
+   * Note: a value of {@code null} is equivalent to {@code NONE}
+   */
```

Comment:
Thank you! :clap: I'll split this into javadoc for the individual fields and methods before merging this in Chunky but this is already better than not having any documentation at all.